activeadmin-mongoid.gemspec

```diff
@@ -20,8 +20,9 @@ Gem::Specification.new do |gem|
   
   gem.required_ruby_version = '>= 2.3'
 
-  gem.add_runtime_dependency 'mongoid',     ['~> 6.0']
   gem.add_runtime_dependency 'activeadmin', '>= 1.3.0'
+  gem.add_runtime_dependency 'mongoid',     ['~> 6.0']
```

Comment:
This is picky me talking... :sweat_smile:. Can you move this line back to where it was, to make the diff smaller, and the history of dependency changes easier to track.

Reply:
no problem! just a couple of minutes to change